zip-zcash-grants-fund.rst

```diff
@@ -0,0 +1,330 @@
+
+3. Priority SHOULD be given to grants that bolster teams with
+   substantial (current or prospective) continual existence, and set them up
+   for long-term success, subject to the usual grant award considerations
+   (impact, ability, risks, team, cost-effectiveness, etc.). Priority SHOULD be
+   given to grants that support ecosystem growth, for example through
+   mentorship, coaching, technical resources, creating entrepreneurial
+   opportunities, etc. If one proposal substantially duplicates another's
+   plans, priority SHOULD be given to the originator of the plans.
+
+4. Grants SHOULD be restricted to furthering the Zcash cryptocurrency and
+   its ecosystem (which is more specific than furthering financial privacy in
+   general) as permitted under Section 501(c)(3).
+
+5. Grant awards are subject to approval by a five-seat Zcash Community
+   Grants Committee. The Zcash Community Grants Committee SHALL be selected by the
+   ZF's Community Advisory Panel or successor process. Elections SHALL be staggered to
+   ensure continuity within the Committee.
+
+6. The Zcash Community Grants Committee's funding decisions will be final, requiring
+   no further approval, but are subject to veto if FPF
+   judges them to violate laws or if FPF reporting requirements and other
+   (current or future) obligations.
+
+7. Zcash Community Grants Committee members SHALL have a one-year term and MAY sit
+   for reelection. The Zcash Community Grants Committee is subject to the same
+   conflict of interest policy that governs the ZF Board of Directors (i.e. they
+   MUST recuse themselves when voting on proposals where they have a financial
+   interest). At most one person with association with the BP/ECC, and at most
+   one person with association with the ZF, are allowed to sit on the Major
+   Grant Review Committee. "Association" here means: having a financial
+   interest, full-time employment, being an officer, being a director, or having
+   an immediate family relationship with any of the above. The ZF SHALL continue
+   to operate the Community Advisory Panel and SHOULD work toward making it more
+   representative and independent (more on that below).
+   
+   Zcash Community Grants Committee members are expected to work approximately 35
+   hours per month and will be compensated accordingly from the Zcash Community Grants Committee
+   budget. The total compensation for the committee is paid from the Zcash Community Grants Committee budget. 
+   This works out to be a total of 175hours of compensation per month which is roughly the equivalent of a 
+   signle full time position.
+   
+8. A portion of the funds shall be allocated to a
+   Discretionary Budget, which may be disbursed for expenses reasonably related
+   to the administration of Zcash Community Grants. The amount of funds allocated to the
+   Discretionary Budget SHALL be decided by the ZF's Community Advisory Panel or

```

Comment:
Just to confirm, ts ZCAP the intended method here, considering that FPF would manage ZCG funds?